install_prerequisites.py

```diff
@@ -89,7 +89,7 @@ def main():
             repo_list_path = '/etc/apt/sources.list.d/goreleaser.list'
 
             if not os.path.exists(repo_list_path):
-                # install apt-transport-https and ca-certificates first
+                # installca-certificates first
```

Comment:
Sorry, couldn't help it...
```suggestion
                # install ca-certificates first
```